app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -995,6 +995,21 @@ class PatientSummaryUpdateTest {
         ))
   }
 
+  @Test
+  fun `when done is clicked and patient is not dead, then load data for done click`() {
+    val model = defaultModel
+        .currentFacilityLoaded(facility)
+        .patientSummaryProfileLoaded(patientSummaryProfile)
+
+    updateSpec
+        .given(model)
+        .whenEvent(PatientSummaryDoneClicked(patientUuid))
+        .then(assertThatNext(
+            hasNoModel(),
+            hasEffects(LoadDataForDoneClick(patientUuid))
+        ))
+  }
+
```

Comment:
Given that we have added some Mobius style tests for existing user flows in `PatientSummaryScreen`, should we add the remaining effect handler tests and then remove the tests pertaining to this flow from the older regression tests?

Reply:
Yeah, I can do that in a different PR?

Reply:
Yup, that works.